src/js/control-bar/progress-control/seek-bar.js

```diff
@@ -167,8 +168,6 @@ class SeekBar extends Slider {
         this.duration_ = duration;
       }
     });
-
-    return percent;
```

Comment:
we should still be returning this value as it may be a breaking change.

Reply:
Because the `update` function is throttled, so even you trigger a  `timeupdate` event manually when finish the seek on `mouseup`, the `update` function only called once at the same tick.

Reply:
The return value of `update` function seems not used.

Reply:
Ah, yeah, the throttling is a good point.

While we may not be using it, if someone who uses Video.js is extending the SeekBar themselve and expecting `update` to return a value, they will break. We should avoid changes that have a potential breaking change in them.

Reply:
Before this follow commit: https://github.com/videojs/video.js/pull/6155
The `update` function does not has a return value, so I think it's not breaking change.

Reply:
Since the `percent` depends on the Slider's `update` function. The return of undefined does not make sense.
https://github.com/videojs/video.js/blob/master/src/js/slider/slider.js#L238-L248

Reply:
I tried to remove the throttling of the `update` function, it works, but may cause a little perf issue. So there's a trade-off.

Reply:
@gkatsev, If you think removing the throttling of the `update` function is more reasonable, I will send another pr.

Reply:
No, I don't think we should unthrottle it. I'll have to think about it a bit.

Reply:
Ok, hope for a better solution.